corgi/api/filters.py

```diff
@@ -164,35 +158,10 @@ class Meta:
         method="filter_gomod_components",
     )
 
-    @staticmethod
-    def filter_re_provides_purl(
-        queryset: ComponentQuerySet, _name: str, value: str
-    ) -> QuerySet["Component"]:
-        """Filter components by purl regex search on provided components."""
-        if value in EMPTY_VALUES:
-            # User gave an empty ?param= so return the unfiltered queryset
-            return queryset
-        return queryset.filter(provides__purl__iregex=value).distinct()
-
-    @staticmethod
-    def filter_provides_name(
-        queryset: ComponentQuerySet, _name: str, value: str
-    ) -> QuerySet["Component"]:
-        """Filter components by name match on provided components."""
-        if value in EMPTY_VALUES:
-            # User gave an empty ?param= so return the unfiltered queryset
-            return queryset
-        return queryset.filter(provides__name=value).distinct()
-
-    @staticmethod
-    def filter_re_provides_name(
-        queryset: ComponentQuerySet, _name: str, value: str
-    ) -> QuerySet["Component"]:
-        """Filter components by name regex search on component provides."""
-        if value in EMPTY_VALUES:
-            # User gave an empty ?param= so return the unfiltered queryset
-            return queryset
-        return queryset.filter(provides__name__iregex=value).distinct()
+    active_streams = BooleanFilter(
```

Comment:
please put this back - ordering was intentional eg. we want active streams to filter before latest_components